### PR TITLE
Allow `Constraint` to convert `Int`s into `Float64` in the bounds

### PR DESCRIPTION
**Pull request by stelmo**
This is just for convenience, because I keep on forgetting to convert it manually... Thoughts?

### CODE REVIEW

File: src/constraint.jl [hunk 1]
```diff
@@ -26,6 +26,11 @@ Base.@kwdef mutable struct Constraint{V}
     end
 end
 
+Constraint(v::T, b::Int) where {T<:Value} = Constraint{T}(v, float(b))
+Constraint(v::T, b::Tuple{Int, Int}) where {T<:Value} = Constraint{T}(v, float.(b))
+Constraint(v::T, b::Tuple{Int, Float64}) where {T<:Value} = Constraint{T}(v, float.(b))
+Constraint(v::T, b::Tuple{Float64, Int}) where {T<:Value} = Constraint{T}(v, float.(b))
```
Comment (exaexa) [inline src/constraint.jl]:
```suggestion
Constraint(v::T, b::Real) where {T<:Value} = Constraint{T}(v, Float64(b))
Constraint(v::T, b::Tuple{A,B}) where {A<:Real,B<:Real} = Constraint{T}(v, Float64.(b))
```

- use exact target datatype
- we've got `Real` :]

Comment (exaexa) [reply src/constraint.jl]: btw this still needs tests, pls just change some of the floats in the docs to ints so that this triggers.

**Review by exaexa — CHANGES_REQUESTED** [review]
small changes needed

File: src/constraint.jl (hunk 1 above)
Comment (stelmo) [reply src/constraint.jl]: fixed!

**Review by exaexa — APPROVED** [review]
k good

**Pull request merged**